packages/contracts/deploy-config/goerli-testnet.ts

```diff
@@ -3,8 +3,6 @@ const config = {
   l1BlockTimeSeconds: 15,
   l2BlockGasLimit: 15_000_000,
   l2ChainId: 5001,
-  updateGaslimitBlock: 222073,
-  eigenDaBlock: 8280000,
   ctcL2GasDiscountDivisor: 32,
```

Comment:
what's the related config now?

Reply:
These two config has been set as below:
- https://github.com/mantlenetworkio/mantle/blob/0b7733c10a44e93edbf49b27e056ebe2e5279c63/l2geth/core/upgrade/config.go#L54
- https://github.com/mantlenetworkio/mantle/blob/0b7733c10a44e93edbf49b27e056ebe2e5279c63/l2geth/core/upgrade/config.go#L55

Reply:
> what's the related config now?

These two config has been set as below:
- https://github.com/mantlenetworkio/mantle/blob/0b7733c10a44e93edbf49b27e056ebe2e5279c63/l2geth/core/upgrade/config.go#L54
- https://github.com/mantlenetworkio/mantle/blob/0b7733c10a44e93edbf49b27e056ebe2e5279c63/l2geth/core/upgrade/config.go#L55